### ISSUE TITLE
Tabbing through buttons doesn't work in Firefox on Mac

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#4573]
**Describe the bug**
As an applicant, when I am applying to a program using the Firefox browser on a Mac computer and navigating the page via tabbing, I cannot reach several navigation points including:
- Previous button
- Review button
- End session
- Log in
- Create account
(there may be others as well)

**To Reproduce**
Using the Firefox browser,

1. Publish a program
2. Log in as a guest user to apply to program
3. Attempt to tab through the page to reach any of the navigation points listed above

**Expected behavior**
Expect to be able to select the "Previous" and "Review" buttons via tabbing (this works in Chrome on Mac and works in Firefox on Linux and Windows)

**Desktop (please complete the following information):**

- OS: macOS (intel and apple silicon)
- Browser: Firefox
- Version: 111.0.1

**Comment by gwendolyngoetz** [comment]
This isn't an issue with CiviForm. It's a Mac OS and Firefox configuration problem. https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/

**Comment by reemaka** [comment]
Should we close this since it's not something we can fix on our end then?
